node-via-nvm/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+NODE_VERSIONS=("${NODE_VERSIONS}")
+BOLD='\033[0;1m'
+CODE='\033[36;40;1m'
+RESET='\033[0m'
+
+printf "$${BOLD}Installing nvm!$${RESET}\n"
+
+export NVM_DIR="${INSTALL_PREFIX}/nvm"
```

Comment:
It's nice to assign tf vars to shell vars so that we always deal with them in the same way, I'd suggest the following change:

```suggestion
# top of file
INSTALL_PREFIX='${INSTALL_PREFIX}'

# here
export NVM_DIR="$${INSTALL_PREFIX}/nvm"
```